src/effects/native/bitcrushereffect.cpp

```diff
@@ -22,7 +22,7 @@ EffectManifest BitCrusherEffect::getManifest() {
     EffectManifestParameter* depth = manifest.addParameter();
     depth->setId("bit_depth");
     depth->setName(QObject::tr("Bit Depth"));
-    depth->setDescription("Adjusts the bit depth of the samples.");
+    depth->setDescription(QObject::tr("Adjusts the bit depth of the samples."));
```

Comment:
Oh, I see. thanks.
I really have hard times to come back to c++, object, etc...